random1on1/api/pairings.py

```diff
@@ -0,0 +1,8 @@
+import random1on1
+
+
+class Pairings(object):
```

Comment:
DOCS!

Will this be some kind of a list of 2-tuples? Maybe we don't need a class for this.

Reply:
I wanted to add human readable serialization and deserialization here and possibly more metadata (eg date of the pairing generation etc).

Just haven't done anything for it yet -- will probably be more clear in a future PR